### ISSUE TITLE
Trouble when installing

### ISSUE THREAD

**Issue by thiendi2402** [bizley/yii2-podium#132]
I'm new to Yii2 and want to add podium to my project, I want to install it via zip file.
I'm stuck at first step, I download the zip file podium 0.7
So do I have to rename it and copy to somewhere? And do I have to change composer.json file?
Then do I have to update main.php of console and frontend?
I've try to change composer.json and 2 main.php file but not download zip file then go to the URL install and this happen. 
![image](https://user-images.githubusercontent.com/34855302/34352913-de362d06-ea57-11e7-98bd-dba540eb7651.png)
If someone can help me I will very appreciated. Thanks

**Comment by bizley** [comment]
Have you followed the [instructions](https://github.com/bizley/yii2-podium/wiki/1.-Installation#optional-installation-from-zip-file)?

**Comment by xu0711** [comment]
i have the same issue too.
i followed the instructions at Composer installation - steps #1;
and modified the common/config/main-local.php like this:
<?php
return [
    'components' => [
        'db' => [
            'class' => 'yii\db\Connection',
            'dsn' => 'mysql:host=localhost;dbname=garden180105',
            'username' => 'root',
            'password' => '',
            'charset' => 'utf8',
        ],
        'mailer' => [
            'class' => 'yii\swiftmailer\Mailer',
            'viewPath' => '@common/mail',
            // send all mails to a file by default. You have to set
            // 'useFileTransport' to false and configure a transport
            // for the mailer to send real emails.
            'useFileTransport' => true,
        ],
    ],

    'bootstrap' => ['log', 'podium'],
    'modules' => [
        'podium' => [
            'class' => 'bizley\podium\Podium',
        ],
    ],
];

then i go to the url:
![image](https://user-images.githubusercontent.com/35283142/34757867-fc2f3d62-f60e-11e7-9eb9-b1efcf0e4d6c.png)

i don't know how to deal with it now.
 can you help me?

**Comment by bizley** [comment]
I've just checked it and it works. Are you sure you followed the instructions correctly? What I did:
1. Downloaded fresh Yii 2 Advanced Template Project through composer,
2. Run `init`,
3. Updated `composer.json` with Podium package,
4. Run `composer update`,
5. Updated `common/config/main-local.php` with Podium configuration like yours,
6. Opened URL `http://localhost/yii-application/frontend/web/index.php?r=podium/install/run`

Installation screen is there:

![podium-instal](https://user-images.githubusercontent.com/8577314/34761615-d9813704-f5e4-11e7-8de9-5af4f5286524.png)
